src/test/java/org/folio/tlib/example/MainVerticleTest.java

```diff
@@ -0,0 +1,139 @@
+package org.folio.tlib.example;
+
+import io.restassured.RestAssured;
+import io.restassured.builder.RequestSpecBuilder;
+import io.restassured.http.ContentType;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import io.vertx.core.DeploymentOptions;
+import io.vertx.core.Vertx;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import io.vertx.ext.unit.TestContext;
+import io.vertx.ext.unit.junit.VertxUnitRunner;
+import org.folio.okapi.common.XOkapiHeaders;
+import org.folio.tlib.postgres.TenantPgPoolContainer;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.testcontainers.containers.PostgreSQLContainer;
+
+import java.io.IOException;
+
+import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.Matchers.is;
+
+@RunWith(VertxUnitRunner.class)
+public class MainVerticleTest {
+  static Vertx vertx;
+  static int port;
+
+  @ClassRule
+  public static PostgreSQLContainer<?> postgresSQLContainer = TenantPgPoolContainer.create();
+
+  @BeforeClass
+  public static void beforeClass(TestContext context) throws IOException {
+    port = 9230;
+    vertx = Vertx.vertx();
+    RestAssured.enableLoggingOfRequestAndResponseIfValidationFails();
+    RestAssured.baseURI = "http://localhost:" + port;
+    RestAssured.requestSpecification = new RequestSpecBuilder().build();
+
+    DeploymentOptions deploymentOptions = new DeploymentOptions();
+    deploymentOptions.setConfig(new JsonObject().put("port", Integer.toString(port)));
+    vertx.deployVerticle(new MainVerticle(), deploymentOptions)
+        .onComplete(context.asyncAssertSuccess());
+
+  }
+
+  @AfterClass
+  public static void afterClass(TestContext context) {
+    vertx.close().onComplete(context.asyncAssertSuccess());
+  }
+
+  void tenantOp(TestContext context, String tenant, JsonObject tenantAttributes, String expectedError) {
+    ExtractableResponse<Response> response = RestAssured.given()
+        .header(XOkapiHeaders.TENANT, tenant)
+        .header("Content-Type", "application/json")
+        .body(tenantAttributes.encode())
+        .post("/_/tenant")
+        .then()
+        .extract();
+
+    if (response.statusCode() == 204) {
+      return;
+    }
+    context.assertEquals(201, response.statusCode());
+    String location = response.header("Location");
+    JsonObject tenantJob = new JsonObject(response.asString());
+    context.assertEquals("/_/tenant/" + tenantJob.getString("id"), location);
```

Comment:
```suggestion
    assertThat(location, is("/_/tenant/" + tenantJob.getString("id")));
```
`.body("titles", hasSize(2))` has the expected value at the end, the other asserts should use the same order.